apps/condo/bin/runCypressLoadTests.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Runs cypress load tests.
+ *
+ * Prepare:
+ * 1. You should be able to run Cypress
+ * 2. You should have metrics set up
+ *
+ * Usage:
+ * yarn node ./runCypressLoadTests <number of Cypress instances> <run continuously?>
+ *
+ * Examples:
+ * yarn node ./runCypressLoadTests 5 continuous // Runs 5 cypress nodes continuously
+ * yarn node ./runCypressLoadTests 5 // Runs 5 cypress nodes once!
+ */
+
+const { spawn } = require('child_process')
+
+const conf = require('@open-condo/config')
+
+const logger = console
+
+/**
+ * Transforms cypress message to the one-liner.
+ * @param {string} message
+ * @returns {*}
+ */
+const trimMessage = (message) => {
+    return message
+        .replaceAll('─', '')
+        .replaceAll( /  +/g, ' ' )
+        .replaceAll('\n', '')
+        .replaceAll('\r\n', '')
+}
+
+
+const processCypressStdoutMessage = (message, instance) => {
+    const msg = trimMessage(message.toString())
+
+    if (msg.includes('Running:')) {
+        logger.info(`STDOUT Cypress instance: ${instance} ${msg}`)
+    }
+}
+
+const processCypressStderrMessage = (message, instance) => {
+    const msg = trimMessage(message.toString())
+
+    logger.warn(`STDERR Cypress instance: ${instance} ${msg}`)
+}
+
+
+const runCypressInParallel = (instances, cmd, args) => {
+    const cypressInstances = []
+
+    for (let i = 0; i < instances; i++) {
+
+        const cypressInstance = spawn(cmd, args)
+
+        cypressInstance.stdout.on('data', (data) => {
+            processCypressStdoutMessage(data, i)
+        })
+
+        cypressInstance.stderr.on('data', (data) => {
+            processCypressStderrMessage(data, i)
+        })
+
+        cypressInstance.on('close', (code) => {
+            logger.info(`Cypress instance ${i} close all stdio with code ${code}`)
+        })
+
+        cypressInstance.on('error', (err) => {
+            logger.error(`Cypress instance ${i} emitted an error: ${err}`)
+        })
+
+        cypressInstances.push(cypressInstance)
+    }
+}
+
+
+const runCypressContinuously = async (instances, cmd, args) => {
+
+    let idx = 0
+    let cypressInstances = {}
+
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
```

Comment:
There is no way to cancel this task? Maybe provide some functionality to prevent infinite loop running?